Platforms/QemuSbsaPkg/PlatformBuild.py

```diff
@@ -220,7 +203,7 @@ def GetLoggingLevel(self, loggerType):
         txt  == plain text file logging
         md   == markdown file logging
         '''
-        return logging.DEBUG
+        return logging.INFO

```

Comment:
The comment above this seems out of date, we should remove it, too.

Reply:
I updated the comment instead of just deleting it.